### PR TITLE
Changes for Copy flags

### PR DESCRIPTION
**Pull request by InTheCloudDan**
Add additional options and fix path for copying flags within a project.

I'm not sure if these are all named appropriately. So please give feedback there also if it's way off.

### CODE REVIEW

File: spec/definitions.yaml [hunk 1]
```diff
@@ -776,4 +776,11 @@ CustomProperty:
...
     - name
-
+FeatureFlagCopyObject:
+  type: object
+  properties:
+    key:
+      type: string
+    version:
+      type: integer
+      example: 65
```
Comment (eli-darkly) [inline spec/definitions.yaml]: Neither of these is optional, right? In that case I think you want to add this:
```
  required:
    - key
    - version
```

Comment (InTheCloudDan) [reply spec/definitions.yaml]: ahh great catch thanks! So key is required, but version is optional.

Comment (eli-darkly) [reply spec/definitions.yaml]: Unfortunately I don't know the answer to the build error you're seeing.

File: spec/parameters.yaml [hunk 2]
```diff
@@ -35,6 +35,36 @@ WebhookPostRequest:
...
+          type: string
+          example: []
+          enum: [updateOn, updatePrerequisites, updateTargets, updateRules, updateFallthrough, updateOffVariation]
+        description: Define the parts of the flag configuration that will be copied.
+      excludedActions:
+        type: array
+        items:
+          type: string
+          example: []
+          enum: [updateOn, updatePrerequisites, updateTargets, updateRules, updateFallthrough, updateOffVariation]
```
Comment (ashanbrown) [inline spec/parameters.yaml]: Maybe we can dry this up into single $ref.

Comment (InTheCloudDan) [reply spec/parameters.yaml]: latest commit cleans this up

**Pull request merged**